tests/topotests/bgp_evpn_mh_l2l3vni_ext_learn/spine1/evpn.conf

```diff
@@ -0,0 +1,18 @@
+frr defaults datacenter
```

Comment:
I see the addition of a bunch of empty files this makes no sense.  Please remove them.  The topotest should be it's own commit too.

Reply:
We also require all new tests to have a integrated config.  Please change it.